docker/package/baking_wizard.py

```diff
@@ -0,0 +1,303 @@
+            help="Options: 'ledger' if you have the key on a ledger, 'secret-key' if you have"
+            "either the unencrypted or password-encrypted secret key for your address, "
+            "'json' if you have a faucet JSON file from https://faucet.tzalpha.net/.",
+            default="ledger",
+            validator=enum_validator(["ledger", "secret-key", "json"]),
+            completer=enum_completer(["ledger", "secret-key", "json"]),
+            reqs=lambda vs: vs["mode"] == "baker",
+        ),
+        Step(
+            id="ledger_url",
+            prompt="Provide the ledger URL for importing the baker secret key stored on a ledger.\n",
+            help="Your ledger URL should something like <ledger://<mnemonic>/ed25519/0'/1'>",
+            validator=required_field_validator,
+            reqs=lambda vs: vs["mode"] == "baker" and vs["key_import_mode"] == "ledger",
+        ),
+        Step(
+            id="secret_key",
+            prompt="Provide either the unencrypted or password-encrypted secret key for your address.\n",
+            help="The format is 'unencrypted:edsk...' for the unencrypted key, or 'encrypted:edesk...'"
+            "for the encrypted key.",
+            validator=required_field_validator,
+            reqs=lambda vs: vs["mode"] == "baker"
+            and vs["key_import_mode"] == "secret-key",
+        ),
+        Step(
+            id="json_filepath",
+            prompt="Provide the path to your downloaded faucet JSON file.\n",
+            help="The JSON file will contain the 'mnemonic' and 'secret' fields.",
+            validator=[required_field_validator, filepath_validator],
+            reqs=lambda vs: vs["mode"] == "baker" and vs["key_import_mode"] == "json",
+        ),
+    ]
+
+    for step in steps:
+        if not step.reqs(values):
+            continue
+
+        readline.set_completer(step.completer)
+
+        validated = False
+        while not validated:
+            try:
+                answer = input(step.prompt).strip()
+            except:
+                print("Exiting Tezos Baking Wizard")
+                return None
+
+            if answer.lower() in ["quit", "exit"]:
+                print("Exiting Tezos Baking Wizard")
+                return None
+            elif answer.lower() == "help":
+                print(step.help)
+            else:
+                if not answer and step.default is not None:
+                    answer = step.default
+
+                try:
+                    step.validate(answer)
+                except ValueError as e:
+                    print("Validation error: ", str(e))
+                else:
+                    validated = True
+                    values[step.id] = answer
+
+    return values
+
+
+class Setup:
+    def __init__(self, config):
+        self.config = config
+
+    # Bootstrapping tezos-node
+    def bootstrap_node(self):
+        if self.config["snapshot"]:
+            proc_call(
```

Comment:
If node directory has some blockchain data, snapshot import will fail with the following error:
```
tezos-node: Error:
              Invalid data directory 'Please provide a clean directory by removing:
            context
            lock
            store'.
```

I think it makes sense to do the following:
1) Check if there is blockchain data in the tezos-node data directory.
2) If there is ask the user whether we should keep existing data or nuke it and import a new snapshot.